staging/src/k8s.io/apiserver/pkg/server/egressselector/egress_selector.go

```diff
@@ -144,7 +144,7 @@ type httpConnectProxier struct {
 	proxyAddress string
 }
 
-func (t *httpConnectProxier) proxy(addr string) (net.Conn, error) {
+func (t *httpConnectProxier) proxy(ctx context.Context, addr string) (net.Conn, error) {
```

Comment:
are we not using the context for http connect at all?

Reply:
Not currently, just had to add it to all implementations of Proxier because of a breaking change in 0.0.21. This PR is intended to only update gRPC, but this change is unavoidable